uint8 id 
geometry_msgs/Vector3 corner1
geometry_msgs/Vector3 corner2
geometry_msgs/Vector3 corner3
geometry_msgs/Vector3 corner4


